s4e-backend/src/main/java/pl/cyfronet/s4e/geoserver/op/request/CreateS3CoverageRequest.java

```diff
@@ -29,18 +47,57 @@
         String name;
         String title;
         String srs = RequestConstants.SRS;
+        String type = RequestConstants.S3_COVERAGE_TYPE;
         String nativeFormat = RequestConstants.S3_STORE_TYPE;
         boolean enabled = true;
+        Metadata metadata;
+    }
+
+    @Value
+    @Builder
+    private static class Metadata {
+        List<Entry> entry;
+    }
+
+    @Value
+    @Builder
+    private static class Entry {
+        @JsonProperty("@key")
+        String key = "time";
+        DimensionInfo dimensionInfo;
+    }
+
+    @Value
+    @Builder
+    private static class DimensionInfo {
+        boolean enabled = true;
+        String presentation = "CONTINUOUS_INTERVAL";
+        String units = "ISO8601";
+        DefaultValue defaultValue;
+    }
+
+    @Value
+    @Builder
+    private static class DefaultValue {
+        String strategy = "MINIMUM";
```

Comment:
I thought about it, and it is fine to be MIINIMUM, even though it could come to mind that MAXIMUM is a better strategy. It could be because max means serving the most recent granule - however, that would open up a possibility to get unlicensed EUMETSAT granules, so better make asking for them explicit - and easier to catch in the gateway.
Anyway, because of that add a comment about this, because we (or sb else) may later forget this reasoning and fall for the licensing problem.